packages/ember-runtime/lib/computed/reduce_computed_macros.js

```diff
@@ -278,6 +278,28 @@ export function mapBy(dependentKey, propertyKey) {
   hamster.get('remainingChores'); // [{name: 'write more unit tests', done: false}]
   ```
 
+  You can also use `@each` in your dependent key, the callback will still use the underlying array:
```

Comment:
@locks - FYI `chores.@each` works properly without deprecation as of https://github.com/emberjs/ember.js/pull/12848 (landed in 2.5.0).